corehq/ex-submodules/casexml/apps/case/tests/test_rebuild.py

```diff
@@ -69,6 +63,13 @@ def _assertListNotEqual(self, l1, l2, msg=None, include_ordering=True):
         else:
             self.fail(msg)
 
+
+class CouchCaseRebuildTest(TestCase, CaseRebuildTestMixin):
+
+    @classmethod
+    def setUpClass(cls):
+        super(CouchCaseRebuildTest, cls).setUpClass()
```

Comment:
ah yes, but i don't think it's supposed to be, i just didn't copy over `delete_all_cases`